src/pages/learn/tailwind/sam-selikoff-interview.mdx

```diff
@@ -0,0 +1,206 @@
+export const meta = {
+  title: `Sam Selikoff on Tailwind CSS`,
```

Comment:
can we dig a little deeper on the "title as summary" for this? The slug as well could use similar attention

Reply:
@joelhooks Renamed and updated title... try now